lib-multisrc/libgroup/src/eu/kanade/tachiyomi/multisrc/libgroup/LibGroupDto.kt

```diff
@@ -266,8 +266,8 @@ class Pages(
 
 @Serializable
 class AuthToken(
-    private val auth: Auth,
-    private val token: Token,
+    private val auth: Auth?,
```

Comment:
```suggestion
    private val auth: Auth? = null,
```
Set default value or will fail if the field is not found